geotrek/outdoor/views.py

```diff
@@ -59,12 +61,28 @@ def dispatch(self, *args, **kwargs):
 
 
 class SiteDelete(MapEntityDelete):
+    template_name = "outdoor/site_confirm_delete.html"
     model = Site
+    success_url = "outdoor:site_list"
 
     @same_structure_required('outdoor:site_detail')
     def dispatch(self, *args, **kwargs):
+        self.sites_pk = self.kwargs['pk'].split(',')
+        self.sites = []
+        for pk in self.sites_pk:
+            site = Site.objects.get(pk=pk)
+            self.sites.append(site)
```

Comment:
You can replace this loop with
```suggestion
            self.sites = Site.objects.filter(pk__in=self.sites_pk)
```